.secrets.baseline

```diff
@@ -110,6 +110,14 @@
         "line_number": 75,
         "type": "Secret Keyword"
       }
+    ],
+    "spec/feature/login_spec.rb": [
```

Comment:
We can't use `pragma allowlist secrets` in this one case since rubocop doesn't permit comments on the same line as `def`